charts/osm/crds/config_multicluster_service.yaml

```diff
@@ -86,6 +86,10 @@ spec:
                         type: integer
                         minimum: 0
                         default: 0
+                      priority:
```

Comment:
Even though this field is added, we aren't leveraging it in this PR, is that right?

Reply:
It's not fully utilized. In this PR we only have priority 0 for local and 1 for remote, as a minimal implementation. We can improve it in the future.